### PR TITLE
Fix modal css for mobile

### LINKED ISSUE #130: Fix Modal centering on Mobile
## Expected Behavior

Modal should be centered on mobile.

## Actual/Current Behavior

It's not centered on some devices/browsers.

## Steps to Reproduce the Problem
  1. Start app.
  2. Click on card.
  3. Observe.

## Pre-Testing TODOs

_What needs to be done before testing._

  1. Change CSS media query for modal.

### If you are not a member of this project, _skip this step_

  1. Click on PokeCard
  2. Check modal positioning.

## Environment

iOS, Safari, Firefox

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Added text align centering for all screen sizes.
2. Item 2

## Purpose
Modal was off the center.

## Approach
Modify media query and test.

Closes #130

### CODE REVIEW

File: src/styles/components/_modal.scss [hunk 1]
```diff
@@ -267,8 +270,12 @@ $modal-padding: 25px;
   }
 
   .modalWindow {
-    width: 725px;
-    height: 450px;
+    // width: 725px;
+    // height: 450px;
```
Comment (BlackCubes) [inline src/styles/components/_modal.scss:274]: COMMENTS!!!!

File: src/styles/components/_modal.scss [hunk 2]
```diff
@@ -308,12 +318,13 @@ $modal-padding: 25px;
...
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
     width: 350px;
     min-height: 650px;
-    margin: 0 auto;
-    top: 15%;
-    left: 20%;
-    transform: translate(-15%, -12%);
+    // margin: 0 auto;
```
Comment (BlackCubes) [inline src/styles/components/_modal.scss:327]: A COMMENT!

File: src/styles/components/_modal.scss (hunk 1 above)
Comment (Lap343) [inline src/styles/components/_modal.scss:274]: Oh no. Connie the rebel. Havin comments there

File: src/styles/components/_modal.scss (hunk 2 above)
Comment (Lap343) [inline src/styles/components/_modal.scss:327]: otro! oh no Connie

**Review by BlackCubes — APPROVED** [review]
Looks gewd. Just remove those comments

**Review by Lap343 — APPROVED** [review]
¡Que bueno PR Connie!

**Pull request merged**